src/compas/robots/__init__.py

```diff
@@ -113,12 +113,54 @@
     LocalPackageMeshLoader
 
 """
-
 from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-
-from .model import *  # noqa: F401 F403
-from .resources import *  # noqa: F401 F403
 
-__all__ = [name for name in dir() if not name.startswith('_')]
+from .model import (
+    Geometry, Box, Cylinder, Sphere, Capsule, MeshDescriptor,
+    Color, Texture, Material, Origin
+)
+from .model import (
+    Joint, ParentLink, ChildLink, Calibration, Dynamics,
+    Limit, Axis, Mimic, SafetyController
+)
+from .model import Link, Inertial, Visual, Collision, Mass, Inertia
+from .model import RobotModel, ToolModel
+
+from .resources import AbstractMeshLoader, DefaultMeshLoader, LocalPackageMeshLoader
+from .resources import GithubPackageMeshLoader
```

Comment:
and here as well...  in general, my vote goes for grid imports on every `__init__.py`

Reply:
by grid imports you mean in brackets and one per line?

Reply:
Yep, exactly